packages/@okta/vuepress-site/conductor.yml

```diff
@@ -1209,6 +1207,28 @@ redirects:
     to: /docs/guides/validate-id-tokens/-/overview/
   - from: /guides/custom-url-domain/overview/index.html
     to: /docs/guides/custom-url-domain/overview/
+  - from: /use_cases/promote-oan-integration/index.html
+    to: https://www.okta.com/integrate/documentation/promotion/
+  - from: /docs/api/resources/
+    to: /docs/reference/
+  - from: /docs/api/resources
+    to: /docs/reference/
+  - from: /docs/reference/api
+    to: /docs/reference/
+  - from: /docs/api/resources/oauth2
+    to: /docs/reference/api/oidc/
+  - from: /docs/api/resources/oauth2.html
+    to: /docs/reference/api/oidc/
+  - from: /docs/api/resources/oauth2/index.html
+    to: /docs/reference/api/oidc/
+  - from: /docs/api/resources/social_authentication.html
+    to: /docs/concepts/social-login/
+  - from: /docs/api/resources/tokens
+    to: /docs/guides/validate-access-tokens/overview/
+  - from: /docs/api/getting_started/error_codes.html
+    to: /docs/reference/error-codes/
+  - from: /use_cases/api_security/
+    to: /docs/concepts/api-access-management/
```

Comment:
/docs/concepts/api-access-management/ gives me a 404.

Reply:
![Screen Shot 2019-06-20 at 9 48 53 AM](https://user-images.githubusercontent.com/492300/59866549-ae1f7d00-9340-11e9-9165-f9da473253b2.png)

hmm....working for me - can you recheck?   
https://developer.okta.com/docs/concepts/api-access-management/

Reply:
Working for me now. Probably made a mistake before.